enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CypherComparisonSupport.scala

```diff
@@ -632,23 +649,23 @@ object CypherComparisonSupport {
 
     def Procs: TestConfiguration = TestScenario(Versions.Default, Planners.Default, Runtimes.ProcedureOrSchema)
 
+    /**
+      * Handy configs for things only supported from 3.3 (not rule) and for checking plans
+      */
+    def OldAndRule: TestConfiguration = Cost2_3 + Cost3_1 + AllRulePlanners
+
+    /**
+      * Configs which support CREATE, DELETE, SET, REMOVE, MERGE etc.
+      */
+    def UpdateConf: TestConfiguration = Interpreted - Cost2_3
+
     /*
     If you are unsure what you need, this is a good start. It's not really all scenarios, but this is testing all
     interesting scenarios.
      */
-    def All: TestConfiguration =
-      TestConfiguration(Versions.V3_4, Planners.Cost, Runtimes(Runtimes.CompiledSource, Runtimes.CompiledBytecode)) +
-        TestConfiguration(Versions.Default, Planners.Default, Runtimes(Runtimes.Interpreted, Runtimes.Slotted)) +
-        TestConfiguration(Versions.V2_3 -> Versions.V3_1, Planners.all, Runtimes.Default) +
-        TestScenario(Versions.Default, Planners.Rule, Runtimes.Default) +
-        TestScenario(Versions.V3_3, Planners.Cost, Runtimes.Default)
+    def All: TestConfiguration = AbsolutelyAll - Procs
```

Comment:
When Alex and I made all configurations explicit (instead of using each other with + and -), we tried to make it easier to understand, what a Configuration actually contains. At least for all the "pre-canned" Configurations offered here.

I think that what we came up with is too verbose and, even though it makes explicit what is used, is not that helpful. I am really unsure which approach is more convenient/error-prone.

Just sharing thoughts here :)

Reply:
I see what you mean. I'm also not sure which the best way is. My thinking here was that if you want to add a new scenario to `AbsolutelyAll`, you don't have to add it to `All` as well. And while `All` and `AbsolutelyAll` are not the best names for something which does not contain all configs, this at least shows what the difference between them are, which was not very obvious before